sdk/easm/Azure.Analytics.Defender.Easm/samples/Sample4_SavedFilters.md

```diff
@@ -0,0 +1,68 @@
+# Use Saved Filters
```

Comment:
The Microsoft style guide requires that headers use sentence-casing rather than the title-casing used here.  Please adjust all headers in markdown files.